src/neptune_scale/api/run.py

```diff
@@ -119,6 +122,8 @@ def __init__(
             creation_time: Custom creation time of the run.
             fork_run_id: If forking from an existing run, ID of the run to fork from.
             fork_step: If forking from an existing run, step number to fork from.
+            log_directory: Base directory in which to store the run's database. Defaults to `.neptune` in the current
```

Comment:
I need some info here on what happens if the path is `abc` vs `/abc`. Is the former one created in CWD? Is the latter treated as absolute? Should be maybe have a test for it?